### DISCUSSION TITLE
Support for block size of 100MB

### DISCUSSION THREAD

**Discussion by tschakravarty** [facebook/CacheLib#281] [General]
I have a requirement to cache block size of 100MB, does cachelib support such large size blocks?

**Comment by haowu14** [comment]
Is the block size you mentioned here the slab size in DRAM, or the region size in NVM BlockCache?

**Comment by pbhandar2** [comment]
Closing this discussion as it has been inactive. If this is still relevant, then please create a new discussion post and we will get back to you as soon as we can. Thank you!
